### ISSUE TITLE
[Resource List Page] As a person experiencing homelessness, I need to view listings by categories so that I can easily find what I am looking for.

### ISSUE THREAD

**Issue by amaliebarras** [CodeForBoulder/upswyng#28]
This page is dependent on issues #38 and #39. The map view, shown below, is a lower priority and dependent on #40.

![listingslist.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/565201200fec5f3e00d1ff86/125516c66e82c728ace21e0d46e49d89378b8dae9ce5b650b01fbcd6395428792c07b7692ac7b62be8112b590c5f47fa450a0e44fcf88c7da6b269708a5919f1c32f0e2a92624cf5f2d75dad0ba821834584b3b98b49c4ee3c20e38a8f482c53.png)

- Resource List Page should have a header that says what category user is in
- Resource List Page should have subcategories in tabs that help the user further navigate the category

- Information to be included on each resource preview in the list: 
   - Resource Name
   - Address
   - Short description
   - who it is for
   - what they offer
   - whether they are open or closed
   - whether user has favorited it or not
   - link to full listing (with hours and description)

- Resource List Page should have a toggle so that users can view the resource list in a map view
- Resources are listed in order of distance from user (if provided)

## Acceptance Criteria
[ ] the main resource header at the top of the listing page should have a background equal to the tile of the resource type on the home page
[ ] the bars located at the bottom of each image should correspond with the color of resource tile on the home page.

**Comment by jacobvenable** [comment]
Closing in favor of smaller issues left on this feature.
